src/ApiHandler.ts

```diff
@@ -480,6 +484,15 @@ export default class ApiHandler {
 			specName: version.specName.toString(),
 			specVersion: version.specVersion,
 			txVersion: version.transactionVersion,
+			// TODO - when unwrap fails, do we want to throw or give an error just for this field?
```

Comment:
This sounds like a @shawntabrizi question to me. From my POV, this field can be `null`, but I'm not sure if that's the correct idiom to accurately model what's happening at the node level or whatever.

Reply:
Active era and current era will only be null if the Staking system hasn't started yet. Throwing an error or assuming it is zero is probably fine here depending on what behavior you want.